activity/awssns/activity.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Amazon SNS - SMS",
+  "version": "0.0.1",
+  "type": "flogo:activity",
+  "description": "Send SMS using Amazon Web Services Simple Notification Services",
+  "author": "Phlippe GABERT <pgabert@tibco.com>",
+  "ref": "github.com/philippegabert/flogo-contrib/blob/awssns/activity/awssns",
```

Comment:
This should be updated to github.com/TIBCOSoftware/flogo-contrib/activity/awssns